flink-yarn/src/main/java/org/apache/flink/yarn/Utils.java

```diff
@@ -359,6 +359,40 @@ public static void addToEnvironment(Map<String, String> environment,
 				StringInterner.weakIntern(val));
 	}
 
+	/**
+	 * Resolve keytab path either as absolute path or relative to working directory.
+	 *
+	 * @param workingDir current working directory
+	 * @param keytabPath configured keytab path.
+	 * @return resolved keytab path, or null if not found.
+	 */
+	public static String resolveKeytabPath(String workingDir, String keytabPath) {
+		String keytab = null;
+		if (keytabPath == null) { // keytab not exist
+			LOG.info("keytab path isn't configured!");
```

Comment:
I think we shouldn't log here, because now all users will always get this log message, even if they don't do anything with Kerberos, that might be confusing.